sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/Embedding.java

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Embedding settings within {@link VectorEmbeddingPolicy}
+ */
+public final class Embedding {
+    @JsonProperty(Constants.Properties.PATH)
+    private String path;
+    @JsonProperty(Constants.Properties.VECTOR_DATA_TYPE)
+    private String vectorDataType;
+    @JsonProperty(Constants.Properties.VECTOR_DIMENSIONS)
+    private Long dimensions;
+    @JsonProperty(Constants.Properties.DISTANCE_FUNCTION)
+    private String distanceFunction;
+
+    /**
+     * Gets the path for the embedding.
+     *
+     * @return path
+     */
+    public String getPath() {
+        return path;
+    }
+
+    /**
+     * Sets the path for the embedding.
+     *
+     * @param path the path for the embedding
+     * @return Embedding
+     */
+    public Embedding setPath(String path) {
+        this.path = path;
+        return this;
+    }
+
+    /**
+     * Gets the data type for the embedding.
+     *
+     * @return vectorDataType
+     */
+    public String getVectorDataType() {
+        return vectorDataType;
+    }
+
+    /**
+     * Sets the data type for the embedding.
+     *
+     * @param vectorDataType the data type for the embedding
+     * @return Embedding
+     */
+    public Embedding setVectorDataType(String vectorDataType) {
```

Comment:
Same comment as for `DistanceFunction` - we probably don't need to make `VectorDataType` as public class.

Reply:
Same to Distance Function, VectorDataType class can be used to set the values when creating a new Embedding. 

```
CosmosVectorEmbedding embedding1 = new CosmosVectorEmbedding(
            "/vector1",
            CosmosVectorDistanceFunction.COSINE.toString(),
            3L,
            CosmosVectorDataType.FLOAT32.toString());
```